views/job-role-list.html

```diff
@@ -1,167 +1,308 @@
+                                                                <label
+                                                                    style="float: right; font-size: 16px; font-weight: 400; font-style: italic; padding-bottom: 1rem;">Sort
+                                                                    By: Location
+                                                                    ({{RoleOrdering.orderBy}})</label>
+                                                                {% endif %}
+                                                                {% if RoleOrdering.fieldName == "capabilityName" %}
+                                                                <label
+                                                                    style="float: right; font-size: 16px; font-weight: 400; font-style: italic; padding-bottom: 1rem;">Sort
+                                                                    By: Capability
+                                                                    ({{RoleOrdering.orderBy}})</label>
+                                                                {% endif %}
+                                                                {% if RoleOrdering.fieldName == "bandName" %}
+                                                                <label
+                                                                    style="float: right; font-size: 16px; font-weight: 400; font-style: italic; padding-bottom: 1rem;">Sort
+                                                                    By: Band
+                                                                    ({{RoleOrdering.orderBy}})</label>
+                                                                {% endif %}
+                                                                {% if RoleOrdering.fieldName == "closingDate" %}
+                                                                <label
+                                                                    style="float: right; font-size: 16px; font-weight: 400; font-style: italic; padding-bottom: 1rem;">Sort
+                                                                    By: Closing Date
+                                                                    ({{RoleOrdering.orderBy}})</label>
+                                                                {% endif %}
+                                                            <form action="/sort" method="post" id="roleOrderingForm">
+                                                                <table
+                                                                    style="width: 100%; margin: 0 auto; text-align: center; margin-top: 1rem; margin-left: 5.5rem;">
+                                                                    <tr style="text-align: left;">
+                                                                        <div class="button-container">
```

Comment:
add id for any clickable button/link that the QA guys might need

Reply:
Fixed